__tests__/extension-requests/extension-requests.test.js

```diff
@@ -919,4 +938,31 @@ describe('Tests the Extension Requests Screen', () => {
     );
     expect(ascSortIconDisplayStyle).toBe('block');
   });
+
+  it('Should show empty message if all extension requests have been addressed', async () => {
+    await page.goto(`${baseUrl}/?order=asc&size=1&q=status%3APENDING`);
+    await page.waitForNetworkIdle();
+
+    extensionRequestsElement = await page.$('.extension-requests');
+    let extensionCardsList = await page.$$('.extension-card');
+
+    for (const card of extensionCardsList) {
+      let approveButton = await card.$('.approve-button');
+      await approveButton.click();
+      await page.waitForTimeout(2500);
+    }
```

Comment:
When clicking on the approve button, what happens (Does it make API call to rds backend or is the behaviour intercepted)

Reply:
If it's intercepted, is there a need to wait for 2500 seconds here?

If not, then how does that work?

Reply:
The button triggers an API call, but we're intercepting it in the tests. However, after the API calls succeeds, an animation is triggered that lasts around 1600 ms. I've reduced the timeout to 1700 ms from 2500 ms.